src/System.Buffers.Primitives/System/Buffers/OwnedPinnedArray.cs

```diff
@@ -6,37 +6,6 @@
 
 namespace System.Buffers
 {
-    public class OwnedNativeMemory : OwnedMemory<byte>
```

Comment:
I actually think these implementations could stay in Experimental. /cc @KrzysztofCwalina ?